src/Cache_Command.php

```diff
@@ -188,6 +191,13 @@ public function flush( $args, $assoc_args ) {
 	 */
 	public function get( $args, $assoc_args ) {
 		list( $key, $group ) = $args;
+
+		$global = \WP_CLI\Utils\get_flag_value( $assoc_args, 'global', '' );
```

Comment:
The flag should act as a boolean, so a better default would be `false`:
```suggestion
		$global = \WP_CLI\Utils\get_flag_value( $assoc_args, 'global', false );
```